Controllers/window_controller.py

```diff
@@ -621,15 +748,17 @@ def dynamic_button_click(self, button_definition):
         split_one = video_timestamp.split("Time: ")
         split_two = split_one[1].split("/")
         video_timestamp = split_two[0]
+        timestamp_text = QTableWidgetItem(video_timestamp)
```

Comment:
An exception is thrown every time the user attempts to activate a button when a video isn't loaded. This does not crash the program, but it should be fixed. At the beginning of the method body, we can add a conditional check to make sure that the video has been loaded. If it hasn't been loaded, we can immediately return. Otherwise, we can continue the method execution.

We can most likely check if a video is loaded by checking the source of the media player. If it is None, then a video has not been loaded. Otherwise, it has been loaded.